modules/dhcp_common/record.rb

```diff
@@ -21,6 +21,10 @@ def subnet_address
       @subnet.network
     end
 
+    def type
+      "record"
+    end
```

Comment:
Probably not needed on the base class -- these are never valid instances (only used as a base class).